### PR TITLE
CNV-4076 Dedicated Resources Release Note

### PR DESCRIPTION
**Pull request by bgaydosrh**
See note under Web Console heading.

Original comments are in https://github.com/openshift/openshift-docs/pull/21182

Opened new PR to make rebasing easier.

Test Build:http://file.bos.redhat.com/bgaydos/042720_2/cnv/cnv_release_notes/cnv-release-notes.html

@ousleyp for peer review and merge and pick to *enterprise-4.4*

### CODE REVIEW

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -66,6 +66,14 @@ Virtualization Validation Program (SVVP) to run Windows Server workloads.
 ** SVVP Certificate Red Hat Enterprise Linux CoreOS 8 workers are named _Red Hat OpenShift
 Container Platform 4 on RHEL CoreOS 8_.
 
+=== Web console
+// CNV-4076 UI view/config dedicated resources
+* A new option to view and configure dedicated resources for virtual machines is
+available in the web console *Virtual Machine Details* view. When you enable dedicated
+resources, you ensure that your virtual machine's workload runs on CPUs that
+are not used by other processes. This improves your virtual machine's performance
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:74]: Previous versions had "This *can* improve VM performance", which I think is important to retain as I don't believe an improvement is guaranteed.

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 2]
```diff
@@ -66,6 +66,14 @@ Virtualization Validation Program (SVVP) to run Windows Server workloads.
 ** SVVP Certificate Red Hat Enterprise Linux CoreOS 8 workers are named _Red Hat OpenShift
 Container Platform 4 on RHEL CoreOS 8_.
 
+=== Web console
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:69]: There is already a 'web console' heading. Please remove.

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 3]
```diff
@@ -66,6 +66,14 @@ Virtualization Validation Program (SVVP) to run Windows Server workloads.
...
+// CNV-4076 UI view/config dedicated resources
+* A new option to view and configure dedicated resources for virtual machines is
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:71]: Suggest revision to make it active:
`You can now view and configure dedicated resources for virtual machines in the *Virtual Machine Details* page.`
It's a web console improvement so we don't need to repeat it here.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 1 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc:74]: Done.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 2 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc:69]: Done.

File: cnv/cnv_release_notes/cnv-release-notes.adoc (hunk 3 above)
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc:71]: Done,

**Pull request merged**